src/pages/workspace/workflows/WorkspaceWorkflowsApproverPage.tsx

```diff
@@ -0,0 +1,203 @@
+
+        Object.entries(policyMembers ?? {}).forEach(([accountIDKey, policyMember]) => {
+            const accountID = Number(accountIDKey);
+            if (isDeletedPolicyMember(policyMember)) {
+                return;
+            }
+
+            const details = personalDetails?.[accountID];
+            if (!details) {
+                Log.hmmm(`[WorkspaceMembersPage] no personal details found for policy member with accountID: ${accountID}`);
+                return;
+            }
+
+            const isOwner = policy?.owner === details.login;
+            const isAdmin = policyMember.role === CONST.POLICY.ROLE.ADMIN;
+
+            let roleBadge = null;
+            if (isOwner || isAdmin) {
+                roleBadge = (
+                    <Badge
+                        text={isOwner ? translate('common.owner') : translate('common.admin')}
+                        textStyles={styles.textStrong}
+                        badgeStyles={[styles.justifyContentCenter, StyleUtils.getMinimumWidth(60), styles.badgeBordered]}
+                    />
+                );
+            }
+
+            const formattedMember = {
+                keyForList: accountIDKey,
+                accountID,
+                isSelected: policy?.approver === details.login,
+                isDisabled: policyMember.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE || !isEmptyObject(policyMember.errors),
+                text: formatPhoneNumber(PersonalDetailsUtils.getDisplayNameOrDefault(details)),
+                alternateText: formatPhoneNumber(details?.login ?? ''),
+                rightElement: roleBadge,
+                icons: [
+                    {
+                        source: UserUtils.getAvatar(details.avatar, accountID),
+                        name: formatPhoneNumber(details?.login ?? ''),
+                        type: CONST.ICON_TYPE_AVATAR,
+                        id: accountID,
+                    },
+                ],
+                errors: policyMember.errors,
+                pendingAction: policyMember.pendingAction,
+            };
+
+            if (policy?.approver === details.login) {
+                approverDetails.push(formattedMember);
+            } else {
+                policyMemberDetails.push(formattedMember);
+            }
+        });
+        return [policyMemberDetails, approverDetails];
+    }, [personalDetails, policyMembers, translate, policy?.approver, StyleUtils, isDeletedPolicyMember, policy?.owner, styles]);
+
+    const sections: MembersSection[] = useMemo(() => {
+        const sectionsArray: MembersSection[] = [];
+
+        if (searchTerm !== '') {
+            const filteredOptions = [...formattedApprover, ...formattedPolicyMembers].filter((option) => {
+                const searchValue = OptionsListUtils.getSearchValueForPhoneOrEmail(searchTerm);
+                return !!option.text?.toLowerCase().includes(searchValue) || !!option.login?.toLowerCase().includes(searchValue);
```

Comment:
This line caused a regression here [Custom name user searched with email id shows no results found](https://github.com/Expensify/App/issues/42576#top)